databases/week2/lesson2_homework_part1.sql

```diff
@@ -0,0 +1,30 @@
+USE hyf_lesson2;
+
+-- Part 1: Working with tasks
+-- Write the following sql queries:
+-- Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO task (id, title, description, created, updated, due_date, status_id) VALUES (default, 'Book flight', 'Vacation.', now(), now(), '2023-03-31 ', 1);
+
```

Comment:
Her you forgot to insert user_id , then this won't create a task as user_id is NOT NULL. Test it pls

Reply:
I was using the wrong file to create the database.
It is fixed now.